### PR TITLE
Create liboqs 0.12.0 release candidate 1

### PR DESCRIPTION
**Pull request by dstebila**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -27,107 +27,84 @@ liboqs can also be used in the following programming languages via language-spec
...
 
-This release updates ML-KEM implementations to their [final FIPS 203](https://csrc.nist.gov/pubs/fips/203/final) versions. This release still includes the NIST Round 3 version of Kyber for interoperability purposes, but we plan to remove Kyber Round 3 in a future release. Additionally, this release adds support for MAYO and CROSS digital signature schemes from [NIST Additional Signatures Round 1](https://csrc.nist.gov/Projects/pqc-dig-sig/round-1-additional-signatures) along with stateful hash-based signature schemes [XMSS](https://datatracker.ietf.org/doc/html/rfc8391) and [LMS](https://datatracker.ietf.org/doc/html/rfc8554). Finally, this release provides formally verified implementations of Kyber-512 and Kyber-768 from [libjade](https://github.com/formosa-crypto/libjade/releases/tag/release%2F2023.05-2).
+This release updates the ML-DSA implementation to the [final FIPS 204](https://csrc.nist.gov/pubs/fips/204/final) version. This release still includes the NIST Round 3 veresion of Dilithium for interoperability purposes, but we plan to remove Dilithium Round 3 in a future release.
```
Comment (SWilson4) [inline RELEASE.md:32]:
```suggestion
This release updates the ML-DSA implementation to the [final FIPS 204](https://csrc.nist.gov/pubs/fips/204/final) version. This release still includes the NIST Round 3 version of Dilithium for interoperability purposes, but we plan to remove Dilithium Round 3 in a future release.
```

File: RELEASE.md [hunk 2]
```diff
@@ -27,107 +27,84 @@ liboqs can also be used in the following programming languages via language-spec
...
-This release continues from the 0.10.1 release of liboqs.
+This release continues from the 0.11.0 release of liboqs.
 
 ### Key encapsulation mechanisms
 
-- Kyber: Added formally-verified portable C and AVX2 implementations of Kyber-512 and Kyber-768 from [libjade](https://github.com/formosa-crypto/libjade/releases/tag/release%2F2023.05-2).
-- ML-KEM: Updated portable C and AVX2 implementations of ML-KEM-512, ML-KEM-768, and ML-KEM-1024 to FIP 203 version.
-- Kyber: Patched ARM64 implementations of Kyber-512, Kyber-768, and Kyber-1024 to work with AddressSanitizer.
+- HQC: Fixed correctness bug in decapsulation. Thank you to Célian Glénaz and Dahmun Goudarzi from Quarkslab for identifying the issue.
+- Kyber: Removed Kyber.
```
Comment (SWilson4) [inline RELEASE.md:49]: Kyber will not be removed until the next release, correct? Seems like this should be reworded.

**Review by SWilson4 — COMMENTED** [review]
I've manually triggered the [extended tests](https://github.com/open-quantum-safe/liboqs/actions/runs/12059902700) and the [provider release tests](https://github.com/open-quantum-safe/oqs-provider/actions/runs/12059925416/job/33629408281) for this branch. For the latter tests, I created a "tracker" branch in provider which differs from the branch for open-quantum-safe/oqs-provider#568 in name only.

File: RELEASE.md (hunk 2 above)
Comment (dstebila) [inline RELEASE.md:49]:
```suggestion
```

File: RELEASE.md [hunk 3]
```diff
@@ -27,107 +27,84 @@ liboqs can also be used in the following programming languages via language-spec
...
+This release updates the ML-DSA implementation to the [final FIPS 204](https://csrc.nist.gov/pubs/fips/204/final) version. This release still includes the NIST Round 3 version of Dilithium for interoperability purposes, but we plan to remove Dilithium Round 3 in a future release.
 
-LMS and XMSS are disabled by default due to the security risks associated with their use in software. See the note on stateful hash-based signatures in [CONFIGURE.md](https://github.com/open-quantum-safe/liboqs/blob/0.11.0/CONFIGURE.md#stateful-hash-based-signatures).
+Deprecation notice
+==================
+
+This will be the last release of liboqs to include Kyber (that is, the NIST Round 3 version of Kyber, prior to its standardization by NIST as ML-KEM in FIPS 203). Applications should switch to ML-KEM (FIPS 203).
+
+The addition of ML-DSA FIPS 204 final version to liboqs has introduced a new signature API which includes a context string parameter. We are considering removing the old version of the API without a context string to streamline the API going forward. Users who have an opinion on this are invited to provide input at https://github.com/open-quantum-safe/liboqs/issues/2001.
```
[the remark this answers is not in the document]
Comment (dstebila) [reply RELEASE.md]: I've updated the wording in 23260bd6b8e50079a8d0a0d5e3439d022fa6ad7c.

**Review by SWilson4 — APPROVED** [review]

File: RELEASE.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (baentsch) [reply RELEASE.md]: Thx!

**Review by baentsch — APPROVED** [review]

**Review by bhess — APPROVED** [review]

**Pull request merged**